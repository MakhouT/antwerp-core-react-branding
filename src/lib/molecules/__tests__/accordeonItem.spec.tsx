import * as React from "react";
import * as renderer from "react-test-renderer";

import * as A from "../..";

describe("AccordeonItem", () => {
    it("Renders default", () => {
        // Act
        const component: any = renderer.create(<A.AccordeonItem header="test" />);
        const tree: renderer.ReactTestRendererJSON = component.toJSON();

        // Assert
        expect(tree).toMatchSnapshot();
    });

    it("Renders default, collapsed", () => {
        // Act
        const component: any = renderer.create(<A.AccordeonItem header="test" collapsed />);
        const tree: renderer.ReactTestRendererJSON = component.toJSON();

        // Assert
        expect(tree).toMatchSnapshot();
    });

    it("Renders default, collapsed and onclick", () => {
        // Act
        const component: any = renderer.create(<A.AccordeonItem header="test" collapsed onClick={() => {/**/ }} />);
        const tree: renderer.ReactTestRendererJSON = component.toJSON();

        // Assert
        expect(tree).toMatchSnapshot();
    });
});
