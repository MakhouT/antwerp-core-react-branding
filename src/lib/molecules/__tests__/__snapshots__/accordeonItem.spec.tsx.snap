// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordeonItem Renders default 1`] = `
<div
  className="m-accordion__tab is-open"
>
  <div
    className="m-accordion__header"
    onClick={[Function]}
  >
    test
  </div>
  <div
    className="m-accordion__content"
  >
    <div
      className="u-margin"
      style={undefined}
    />
  </div>
</div>
`;

exports[`AccordeonItem Renders default, collapsed 1`] = `
<div
  className="m-accordion__tab is"
>
  <div
    className="m-accordion__header"
    onClick={[Function]}
  >
    test
  </div>
  <div
    className="m-accordion__content"
  >
    <div
      className="u-margin"
      style={undefined}
    />
  </div>
</div>
`;

exports[`AccordeonItem Renders default, collapsed and onclick 1`] = `
<div
  className="m-accordion__tab is"
>
  <div
    className="m-accordion__header"
    onClick={[Function]}
  >
    test
  </div>
  <div
    className="m-accordion__content"
  >
    <div
      className="u-margin"
      style={undefined}
    />
  </div>
</div>
`;
